Rick and Morty/ViewController/EpisodeDetailViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  EpisodeDetailViewController.swift
+//  Rick&Monty
+//
+//  Created by James Ryu on 2021-12-09.
+//
+
+import UIKit
+
+class EpisodeDetailViewController: UIViewController, SetupTheViewController {
+    
+    /// Network manager managing all the data fetching
+    let networkManager = NetworkingManager.shared
+    /// Array of CharacterResults type that stores all the character information.
+    var characterArray: [CharacterResults] = []
+    /// Identifier needed to segue to EpisodeDetailViewController
+    static let segueIdentifier: String = "episodeDetail"
+    /// Receive the Results data
+    var resultsData: Results?
+    /// I consider this as a container to add another UIViewController block
+    let topView = UIView()
+    /// This is the container where the EpisodeInfoVC will go in
+    let itemViewOne = UIView()
+    /// This is the section where EpisodeCastVC will go in
+    let itemViewTwo = UIView()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        fetchCharacters(ids: characterIDs(addresses: resultsData!.characters))
```

Comment:
The app will crash if `resultsData` is nil. Better to wrap this in an `if let` and show an alert if the data is a nil
```suggestion
        if let resultsData = resultsData {
            fetchCharacters(ids: characterIDs(addresses: resultsData.characters))
        }
```